vpd-manager/manager.cpp

```diff
@@ -696,10 +705,13 @@ void Manager::deleteFRUVPD(const sdbusplus::message::object_path path)
     string chipAddress =
         jsonFile["frus"][vpdFilePath].at(0).value("pcaChipAddress", "");
 
-    // Unbind the LED driver for this FRU
-    cout << "Unbinding device- " << chipAddress << endl;
-    executeCmd(createBindUnbindDriverCmnd(chipAddress, "i2c", "leds-pca955x",
-                                          "/unbind"));
+    if (!chipAddress.empty())
+    {
+        // Unbind the LED driver for this FRU
+        cout << "Unbinding device- " << chipAddress << endl;
+        executeCmd(createBindUnbindDriverCmnd(chipAddress, "i2c",
```

Comment:
Why not this condition check if(!address.empty()) present in other places too, where you call createBindUnbindDriverCmnd(). say at line 603, etc.

Reply:
true, we should have that check to avoid unnecessary executeCmd calls.
But changing that much code for this commit is not logical. I will revert this as well.

Reply:
when a command with empty devAddress is executed, whether it will be ignored or any error will be thrown?

Reply:
It will just continue the execution, with an error on journal that no such device is found.